Mathlib/Data/Real/GoldenRatio.lean

```diff
@@ -90,6 +90,10 @@ theorem gold_sub_goldConj : φ - ψ = Real.sqrt 5 := by
   ring
 #align gold_sub_gold_conj gold_sub_goldConj
 
+theorem gold_exp (n : ℕ) : φ ^ (n + 2) - φ ^ (n + 1) = φ ^ n := by
+  rw [goldenRatio]; ring_nf; norm_num; ring_nf
+#align gold_exp gold_exp
```

Comment:
If you are adding new theorems you shouldn't add the `align` line (don't worry about it if you don't know what it does, it is related to an old version of Lean).